src/PresentationalComponents/IconInline/IconInline.js

```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+import propTypes from 'prop-types';
+import InfoIcon from '../../Icons/InfoIcon';
+import WarningIcon from '../../Icons/WarningIcon';
+import CriticalIcon from '../../Icons/CriticalIcon';
+import './IconInline.scss';
+
+const checkState = state => {
```

Comment:
I don't really like switches and it might be better to pass reference to the component function instead of component. It's also better to use string literal over concatenation.
```JSX
const checkState = {
  warning: WarningIcon,
  info: InfoIcon,
  critical: CriticalIcon
}

export const IconInline = ({ message, state, systemInventory }) => {
  const Icon = checkState[state] || React.Fragment;
   return (
      <div className={ `ins-c-dashboard__info-inline ${systemInventory ? 'ins-m-padding-top ' : ''}` }>
          <Icon />
          <p>{ message }</p>
      </div>
  );
};
```

Reply:
Great suggestion, thank you!